.github/workflows/publish.yml

```diff
@@ -4,23 +4,36 @@ on:
     branches:
       - main
 jobs:
-  npm-publish:
-    name: npm-publish
+  publish:
+    name: publish
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@main
-    - name: Set up Node.js
-      uses: actions/setup-node@main
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+  
+    - uses: actions/setup-node@v1
       with:
         node-version: 14.x
-    - run: yarn install
-    - run: yarn tsc
-    - run: yarn build
-    - run: |
-        git config --global user.email "bot@roadie.io"
+        registry-url: 'https://npm.pkg.github.com/'
```

Comment:
Why publish to github out of curiosity ?

Reply:
Oh, this shouldn't be here, good catch. I was looking at some solutions so this is leftover. However, I will decline this PR as we won't be doing this now @iain-b